plugins/color_panel/macos/FLEColorPanelPlugin.mm

```diff
@@ -33,7 +33,12 @@ - (NSString *)channel {
 - (void)handleMethodCall:(FLEMethodCall *)call result:(FLEMethodResult)result {
   BOOL handled = YES;
   if ([call.methodName isEqualToString:@(plugins_color_panel::kShowColorPanelMethod)]) {
-    [self showColorPanel];
+    BOOL showAlpha = YES;
+    if ([call.arguments isKindOfClass:[NSDictionary class]]) {
```

Comment:
If this isn't true then the call is malformed, so I'd prefer that if the check fails the code NSLog an error message, then call `result(FLEMethodNotImplemented);` and return early.